src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx

```diff
@@ -157,3 +156,26 @@ const IdentitySection = styled.div`
 export const Value = styled.p`
   margin-right: 0.5rem;
 `;
+
+export const BoxedLink = styled(NeutralLink)<
+  Pick<BoxedLinkProps, "primary" | "status">
+>`
```

Comment:
Hm, you use `Pick`, but you picked every keys from `BoxedLinkProps `. What did you want to do?

Reply:
`Pick` is no longer useful with this refacto, fixed 75920d6